packages/cli-hermes/src/profileHermes/downloadProfile.ts

```diff
@@ -69,7 +76,7 @@ export async function downloadProfile(
     // If --raw, pull the hermes profile to dstPath
     if (raw) {
       execSyncWithLog(
-        `adb shell run-as ${packageName} cat cache/${file} > ${dstPath}/${file}`,
+        `adb shell run-as ${packageNameWithSuffix} cat cache/${file} > ${dstPath}/${file}`,
```

Comment:
Do we need to run `as` with suffix? We're not doing this in other `adb` calls in this file.

Reply:
That's a good question. I attempted to modify as little as possible. The prior command was `run-as` so I just used that

Reply:
What I am saying is that on this line: https://github.com/react-native-community/cli/pull/1386/files#diff-13ca0a162291646faf5ecefc657d151b45ebf950563c9f32d38ec39ede1341b1R90 we are not using `packageNameWithSuffix`.

I guess as per your comment, that has to be modified.